lib/conversion.ts

```diff
@@ -35,3 +35,59 @@ export function convertedObjectIDInSearch(
 
   this.sendEvent("conversion", params as InsightsEvent);
 }
+
+export interface InsightsSearchConversionObjectIDsEvent {
+  eventName: string;
+  userID: string;
+  timestamp: number;
+  index: string;
+
+  objectIDs: (string | number)[];
+}
+/**
+ * Sends a conversion report using objectIDs
+ * @param params InsightsSearchConversionObjectIDsEvent
+ */
+export function convertedObjectIDs(
+  params: InsightsSearchConversionObjectIDsEvent
+) {
+  if (!params) {
+    throw new Error(
+      "No params were sent to convertedObjectIDs function, please provide `objectIDs` to be reported"
+    );
+  }
+  if (!params.objectIDs) {
+    throw new Error(
+      "required objectIDs parameter was not sent, conversion event can not be properly sent without"
+    );
+  }
+
+  this.sendEvent("conversion", params as InsightsEvent);
+}
+
+export interface InsightsSearchConversionFiltersEvent {
```

Comment:
Why do we have the `Search` inside the name? This event is not tied to a search right?